scripts/controller/aboutController.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+var aboutController = {};
+
+aboutController.showAbout = function() {
+  $('.page-section').hide();
```

Comment:
Nice job accounting for all of your "page" sections at once by using the class selector!